File: level68/homework/codwars1.py
#https://www.codewars.com/kata/555eded1ad94b00403000071/train/python
def series_sum(n):
    sum = 0.0
    for i in range(0,n):
        sum += 1 / (1 + 3 * float(i))
    return '%.2f' % sum

File: level 056/homework/codwars1.js
// /https://www.codewars.com/kata/57f780909f7e8e3183000078/train/javascript
const grow = x => {
  let res = 1;
  for (let i = 0; i < x.length; i++) {
    res *= x[i];
  }
  return res;
};